site/pages/view-all-disruptions.page.tsx

```diff
@@ -403,23 +444,69 @@ const ViewAllDisruptions = ({ disruptions, services, newDisruptionId }: ViewAllD
                         <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
                     </svg>
                 </Link>
-                <button
-                    className="govuk-button govuk-button--secondary block"
-                    data-module="govuk-button"
-                    onClick={() => {
-                        if (showFilters) {
-                            setShowFilters(false);
-                        } else {
-                            setShowFilters(true);
-                        }
-                    }}
-                >
-                    {showFilters ? "Hide and clear f" : "F"}ilters
-                </button>
+                <div className="flex">
+                    <button
+                        className="govuk-button govuk-button--secondary block"
+                        data-module="govuk-button"
+                        onClick={() => {
+                            if (showFilters) {
+                                setShowFilters(false);
+                            } else {
+                                setShowFilters(true);
+                            }
+                        }}
+                    >
+                        {showFilters ? "Hide " : "Show "}filters
```

Comment:
If filters are added and then hide the filters, followed by clicking show filters - the table displays filtered result but the filter values are not displayed in their respective fields